spec/features/event/new_event_spec.rb

```diff
@@ -48,6 +49,52 @@
     end
   end
 
+  context "for ranking lists" do
+    before :each do
+      visit new_rankinglist_path
+    end
+
+    it "should show a field for choosing the ranking metric" do
+      expect(page).to have_field('rankinglist_game_mode')
+    end
+
+    it "should show a field for choosing the initial value of the ranking metric" do
+      expect(page).to have_field('event_initial_value')
+    end
+
+    it "should not show a field for defining a deadline" do
+      expect(page).not_to have_field('event_deadline')
+    end
+
+    it "should not show a field for defining a enddate " do
+      expect(page).not_to have_field('event_enddate')
+    end
+
+    it "should not show a field for defining a startdate" do
+      expect(page).not_to have_field('event_startdate')
+    end
+
+    it "should not show a field for defining whether its a team or single player sport" do
+      expect(page).not_to have_field('event_player_type')
+    end
+
+    it "should not show a field for the duration" do
+      expect(page).not_to have_field('event_duration')
+    end
+
+    it "should be possible to create a rankinglist" do
```

Comment:
This test contains assumptions about the validity of events. Using a factory to fill in the fields could solve that.